src/App/pinJSONAndImage/pin.service.ts

```diff
@@ -40,7 +41,7 @@ class PinService {
   }
 
   async pinJSON(body: string): Promise<IpfsHash | any> {
-    const data = JSON.parse(`${body}`)
+    const data: ValidWiki = JSON.parse(`${body}`)
```

Comment:
is there any reason to cast a string to string? 😕 

```suggestion
    const data: ValidWiki = JSON.parse(body)
```

Reply:
if its not a string typescript wont let him pass a function param in : pinJSON(body: string)

so its checking the type string in the function param, then doing some weird cast to string using literals. Not big deal but I am fairly confident that can be simplified w the suggestion. Let's not stop this merge but take a look if you want later 👍🏻

Reply:
oh okay, I see what you mean ...I'll look at it later 🤔